control/controlknobs/controlknobs.go

```diff
@@ -44,6 +44,9 @@ type Knobs struct {
 	// incremental (delta) netmap updates and should treat all netmap
 	// changes as "full" ones as tailscaled did in 1.48.x and earlier.
 	DisableDeltaUpdates atomic.Bool
+
+	// PeerMTUEnable is whether the node should do peer path MTU discovery.
+	PeerMTUEnable syncs.AtomicValue[opt.Bool]
```

Comment:
Can I ask about using `syncs.AtomicValue[opt.Bool]` over `atomic.Bool`? It seems like we only ever set it to true and we write it back to `*Knobs` unconditionally, so I'm not sure what the false represents (as opposed to unset or true?)

Reply:
Good catch, that's left over from a previous implementation! Thank you. FYI the plan is to replace this with a corresponding PeerMTUDisable when we start enabling it by default.